tests/topotests/fpm_testing_topo1/test_fpm_topo1.py

```diff
@@ -57,14 +57,15 @@ def setup_module(module):
         router.load_config(
             TopoRouter.RD_ZEBRA,
             os.path.join(CWD, "{}/zebra.conf".format(rname)),
-            "-M dplane_fpm_nl",
+            "-M dplane_fpm_nl --asic-offload=notify_on_offload",
         )
         router.load_config(
             TopoRouter.RD_SHARP, os.path.join(CWD, "{}/sharpd.conf".format(rname))
         )
         router.load_config(
             TopoRouter.RD_FPM_LISTENER,
             os.path.join(CWD, "{}/fpm_stub.conf".format(rname)),
+            "-r"
```

Comment:
looks like there's a style warning about this line?

Reply:
yeah let me fix